tst/SignalingApiFunctionalityTest.cpp

```diff
@@ -411,7 +411,7 @@ TEST_F(SignalingApiFunctionalityTest, mockViewer)
     EXPECT_FALSE(IS_VALID_SIGNALING_CLIENT_HANDLE(signalingHandle));
 }
 
-TEST_F(SignalingApiFunctionalityTest, invalidChannelInfoInput)
+TEST_F(SignalingApiFunctionalityTest, DISABLED_invalidChannelInfoInput)
```

Comment:
These are valuable tests and I wouldn't disable them. The "flakiness" is coming from TravisCI networking. We should consider doing something about the test bed rather disabling this but for now, if this fails, re-run the tests

Reply:
I'm just disabling for debugging why travis run fails. It's crashing when running this test.